docker-run-sync-rapid-pro-to-engagement-db.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
can you chmod +x this file please so I can $ ./docker-run...

Reply:
updated.